frigate/genai/__init__.py

```diff
@@ -36,39 +37,53 @@ def __init__(self, genai_config: GenAIConfig, timeout: int = 60) -> None:
-
-        Here is information already known:
-        - Activity occurred at {review_data["timestamp"].strftime("%I:%M %p")}
-        - Detected objects: {review_data["objects"]}
-        - Recognized objects: {review_data["recognized_objects"]}
-        - Zones involved: {review_data["zones"]}
-
-        Your response **MUST** be a flat JSON object with:
-        - `scene` (string): A full description including setting, entities, actions, and any plausible supported inferences.
-        - `confidence` (float): A number 0–1 for overall confidence in the analysis.
-        - `potential_threat_level` (integer, optional): Include only if there is a clear, observable security concern:
-            - 0 = Normal activity is occurring
-            - 1 = Unusual but not overtly threatening
-            - 2 = Suspicious or potentially harmful
-            - 3 = Clear and immediate threat
-
-        **IMPORTANT:**
-        - Values must be plain strings, floats, or integers — no nested objects, no extra commentary.
+Please analyze the image(s), which are in chronological order, strictly from the perspective of the {review_data["camera"].replace("_", " ")} security camera.
+
+Your task is to provide a **neutral, factual, and objective description** of the scene, while also:
+- Clearly stating **what is happening** based on observable actions and movements.
+- Including **reasonable, evidence-based inferences** about the likely activity or context, but only if directly supported by visible details.
+
+When forming your description:
+- **Facts first**: Describe the time, physical setting, people, and objects exactly as seen.
+- **Then context**: Briefly note plausible purposes or activities (e.g., “appears to be delivering a package” if carrying a box to a door).
+- Clearly separate certain facts (“A person is holding an object with horizontal rungs”) from reasonable inferences (“likely a ladder”).
+- Do not speculate beyond what is visible, and do not imply hostility, criminal intent, or other strong judgments unless there is unambiguous visual evidence.
+
+Here is information already known:
+- Activity occurred at {review_data["timestamp"].strftime("%I:%M %p")}
+- Detected objects: {review_data["objects"]}
+- Recognized objects: {review_data["recognized_objects"]}
+- Zones involved: {review_data["zones"]}
+
+Your response **MUST** be a flat JSON object with:
+- `scene` (string): A full description including setting, entities, actions, and any plausible supported inferences.
+- `confidence` (float): A number 0-1 for overall confidence in the analysis.
+- `potential_threat_level` (integer, optional): Include only if there is a clear, observable security concern:
+    - 0 = Normal activity is occurring
+    - 1 = Unusual but not overtly threatening
+    - 2 = Suspicious or potentially harmful
+    - 3 = Clear and immediate threat
+{other_concerns}
+
+**IMPORTANT:**
+- Values must be plain strings, floats, or integers — no nested objects, no extra commentary.
```

Comment:
Since this prompt won't be customizable, we probably should find a way to allow the user to specify the language of the response